pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -84,7 +84,7 @@ This can be done in two ways:
 ```
 BALENA_HOST_CONFIG_dtoverlay = pi3-miniuart-bt
 ```
-If you can't find the where to add this configuration go to this page on your dashboard: dashboard.balena-cloud.com/apps/`APP_ID`/config but replace `APP_ID` with the number of your application.
+If you can't find the where to add this configuration go to this page on your dashboard: dashboard.balena-cloud.com/apps/`APP_ID`/config but replace `APP_ID` with the number of your fleet.
```

Comment:
This is crazy! We're suggesting that if they can't find the page to manually construct a URL and visit that.

```suggestion
```